modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -299,7 +299,7 @@ return view.extend({
 
 		o = s.taboption('general', form.Flag, 'rebind_protection',
 			_('Rebind protection'),
-			_('Discard upstream RFC1918 responses.'));
+			_('Discard upstream responses containing <a href="https://datatracker.ietf.org/doc/html/rfc1918" target="_blank" rel="noreferrer noopener">RFC1918</a> addresses.'));
```

Comment:
Can't we take the whole html stuff thing out of translation and just write the following?

```suggestion
                        _('Discard upstream <%s>RFC1918<%s> responses.').format('a href="https://datatracker.ietf.org/doc/html/rfc1918" target="_blank" rel="noreferrer noopener"', '/a'));
```

Everything else is very error-prone from my point of view

Reply:
Absolutely right. Great suggestion. Thanks for the formatting idea.